configure/RELEASE

```diff
@@ -1,4 +1,5 @@
 # RELEASE - Location of external support modules
+MOTOR_ZABER_MOTION = $(MOTOR)/modules/motorZaberMotion
```

Comment:
> I also needed to correct the path to MOTOR_ZABER_MOTION in motorZaberMotion/zaberMotionSupport/configure/RELEASE, which is not something I normally expect to do

Does it make sense to have the user define `MOTOR_ZABER_MOTION` in `configure/RELEASE`, and give the user the option to override that in their own custom RELEASE.local file? It's definitely not an external support module, but if users are used to modifying these files then this might be the right place for it.

Reply:
Ok I just tested building this as standalone and as a motor submodule and it seems to be working. Support libs are being installed in the correct place and standalone configuration is as simple as creating a `RELEASE.local` file.